tests/v2/validation/rbac/README.md

```diff
@@ -36,4 +36,13 @@ rbacInput:
   role: "cluster-owner"
   username: "<userID>"
   password: "<password>"
+```
+
+# Provisioning new clusters
+If the RBAC test needs to provision a new cluster and you don't want to affect other pipeline tests, you should add provisioning.Disable Update Config function on your SetupSuite test, follow the example bellow:
+```go
+func (ra *RestrictedAdminTestSuite) SetupSuite() {
```

Comment:
This might cause some confusion by giving the impression that changes are needed in the config.yaml file to be made by the user which is not the case here. Instead of adding this to the README, can we move this info directly to the test where the function is used. Something like: 

// Disabling configuration here is to avoid interference with other pipeline tests.
// Updates to the config are temporarily disabled during the test
// and are automatically enabled during cleanup.
provisioning.DisableUpdateConfig(ra.client)